draft-ietf-oauth-device-flow.xml

```diff
@@ -16,10 +16,10 @@
 <?rfc compact="yes"?>
 <?rfc subcompact="no"?>
 
-<rfc ipr="trust200902" docName="draft-ietf-oauth-device-flow-03" category="std">
+<rfc ipr="trust200902" docName="draft-ietf-oauth-device-flow-04" category="std">
 
   <front>
-  <title abbrev="OAuth 2.0 Device Flow">OAuth 2.0 Device Flow</title>   
+  <title abbrev="OAuth 2.0 Device Flow">OAuth 2.0 for Browserless and Input Constrained Devices</title>   
```

Comment:
How about: 
OAuth 2.0 Device Flow for Browserless and Input Constrained Devices?

Some background on this change: I wanted to call out that this isn't for *all* devices. Some devices (like smartphones) have better options.